### PR TITLE
1450 rename gossiplb to temperedlb

### LINKED ISSUE #1450: Change name from GossipLB to TemperedLB
**What Needs to be Done?**

Change the name to match the new name from the paper.

### PR DESCRIPTION
**Pull request by nlslatt**
Closes #1450.

### CODE REVIEW

File: docs/md/lb-manager.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ To print LB specification during startup, use `--vt_lb_show_spec` command line f
 | RotateLB       | Testing                 | Rotate objects in a ring                       | `vt::vrt::collection::lb::RotateLB` |
 | RandomLB       | Testing                 | Randomly migrate object with seed              | `vt::vrt::collection::lb::RandomLB` |
 | GreedyLB       | Centralized             | Gather to central node apply min/max heap      | `vt::vrt::collection::lb::GreedyLB` |
-| GossipLB       | Distributed             | Gossip-based protocol for fully distributed LB | `vt::vrt::collection::lb::GossipLB` |
+| TemperedLB     | Distributed             | Gossip-based protocol for fully distributed LB | `vt::vrt::collection::lb::TemperedLB` |
```
Comment (nlslatt) [inline docs/md/lb-manager.md:59]: @ppebay @lifflander @PhilMiller This is the only remaining reference to gossip.  How should we concisely describe `TemperedLB` now?

Comment (lifflander) [reply docs/md/lb-manager.md:59]: Maybe `inspired by epidemic algorithms` or something like that?

**Review by lifflander — APPROVED** [review]
Looks good to me!

**Review by JacobDomagala — APPROVED** [review]

File: docs/md/lb-manager.md (hunk 1 above)
Comment (nlslatt) [reply docs/md/lb-manager.md]: Sounds good to me.  Change made.

**Pull request merged**